cpiface/bess_control.h

```diff
@@ -291,17 +297,13 @@ class BessClient {
     _fseid->set_value_int(fseid);
 
     /* SET VALUES */
-    /* set tunnel action */
-    bess::pb::FieldData *_tunnel = emcaa->add_values();
-    _tunnel->set_value_int(tunnel);
-
-    /* set drop action */
-    bess::pb::FieldData *_drop = emcaa->add_values();
-    _drop->set_value_int(drop);
-
-    /* set notify_cp */
-    bess::pb::FieldData *_notify_cp = emcaa->add_values();
-    _notify_cp->set_value_int(notify_cp);
+    /* set FAR action */
+    uint8_t action = 0;
+    if (tunnel)    action = action | DO_TUNNEL;
+    if (drop)      action = action | DO_DROP;
+    if (notify_cp) action = action | DO_NOTIFY;
```

Comment:
Can there be more than one of these? Should we set or update?

Reply:
I will have to check the specs or discuss this with the entire team.